src/pkg/packager/publish.go

```diff
@@ -123,5 +130,10 @@ func (p *Packager) Publish(ctx context.Context) (err error) {
 			return err
 		}
 	}
+	l.Info("packaged successfully published",
```

Comment:
Usually we print the success logs at debug, but this one is a network request that could take some time, so I opted to make it an info so users get some feedback. Not much happens after this so I could go either way.

Reply:
I think info here makes sense, it gives the URL on the last line of logs which is nice since that's what the user will likely look for.  I'd prefer this line in `remote.PublishPackage` since `remote.PublishPackage` is called elsewhere. (If you run `zarf package create --output oci://registry`), but we have to use the printf format until we switch off old logs.

Reply:
Talked synchronously, good with this here